stories/setup/webpack.config.babel.js

```diff
@@ -1,58 +1,24 @@
-import 'core/polyfill';
+import { getPlugins, getRules } from '../../webpack-common';
+import webpackConfig, {
+  babelOptions as defaultBabelOptions,
+} from '../../webpack.dev.config.babel';
 
-import path from 'path';
-
-import config from 'config';
-import autoprefixer from 'autoprefixer';
-import webpack from 'webpack';
-
-import { getClientConfig } from 'core/utils';
+// We need to add this extra plugin to get Flow to work.
+const babelOptions = {
+  ...defaultBabelOptions,
+  plugins: 'babel-plugin-react-docgen',
```

Comment:
maybe I should create it's own babelrc file here to make it more flexible or what do you think?